content/docs/components-and-props.md

```diff
@@ -156,11 +155,11 @@ function Comment(props) {
 
 [](codepen://components-and-props/extracting-components)
 
-It accepts `author` (an object), `text` (a string), and `date` (a date) as props, and describes a comment on a social media website.
+Acepta `author` (un objeto), `text` (una cadena), y `date` (una fecha) como propiedades, y representa un comentario en una web de redes sociales.
 
-This component can be tricky to change because of all the nesting, and it is also hard to reuse individual parts of it. Let's extract a few components from it.
+Este componente puede ser dificil de cambiar por todo el anidamiento, y tambien es dificil reusar partes individuales de el. Vamos a extraer algunos componentes de este.
```

Comment:
I think it's better to use
> de cambiar debido a todo el anidamiento

instead of 
>de cambiar por todo el anidamiento

And by the end of the sentence, I would change
> componentes de este.

by 
> componentes del mismo.